pkg/sql/set_cluster_setting.go

```diff
@@ -203,9 +203,24 @@ func (n *setClusterSettingNode) startExec(params runParams) error {
 					// hasn't run yet, we can't update the version as we don't
 					// have good enough information about the current cluster
 					// version.
-					return errors.New("no persisted cluster version found, please retry later")
+					if params.extendedEvalCtx.Codec.ForSystemTenant() {
+						return errors.New("no persisted cluster version found, please retry later")
+					}
+					// The tenant cluster in 20.2 did not ever initialize this value and
```

Comment:
When 21.1 is out and we start a fresh multi-tenant cluster then, won't tenants *also* not have this entry, but now they should assume they have 21.1 in there? How will that work? Shouldn't the tenant assume its binary version here?